model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -32,7 +32,7 @@ import org.ossreviewtoolkit.utils.stripCredentialsFromUrl
 
 /**
  * A configuration for a specific package and provenance. It allows to setup [PathExclude]s and
- * [LicenseFindingCuration]s, similar to how its done via the [RepositoryConfiguration] for projects.
+ * [LicenseFindingCuration]s, similar to how it's done via the [RepositoryConfiguration] for projects.
```

Comment:
Nit: In docs we usually do not use contractions like "it's" but spell it out as "it is".